dts/arm/st/f7/stm32f756Xg.dtsi

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright (c) 2018 AJ Palmer
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <st/f7/stm32f746Xg.dtsi>
```

Comment:
Even if this works for now, this is not correct. A package should include a chip, not another SoC's package.
So here, we should have:
```
#include <st/f7/stm32f756Xg.dtsi>
```
And, in st/f7/stm32f756Xg.dtsi:
```
#include <st/f7/stm32f746.dtsi>
```

Reply:
OK, thanks.
I've based the soc section on the 769 - hopefully this corrects this.